src/components/molecules/PlanRoom/RoomTables/Table/index.tsx

```diff
@@ -8,11 +8,14 @@ import { TypeItem } from 'model/tableDND/item-type';
 import { TableType } from 'model/tableDND/table-types';
 import SaveBoard from 'components/molecules/PlanRoom/SaveRoomTables/SaveBoard';
 import { theme } from 'others/theme';
+import basket from 'assets/images/basketDeleteTable.png';
```

Comment:
In my opinion `chakra-ui/icons` and `react-icons` provide all required icons. You don'y need to import images / icons from external sources.

Reply:
will be corrected in the latest version